instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/core/RouteAlternativesTest.kt

```diff
@@ -0,0 +1,81 @@
+package com.mapbox.navigation.instrumentation_tests.core
+
+import androidx.test.espresso.Espresso
+import com.mapbox.api.directions.v5.models.DirectionsRoute
+import com.mapbox.navigation.base.options.NavigationOptions
+import com.mapbox.navigation.base.route.RouteAlternativesOptions
+import com.mapbox.navigation.core.MapboxNavigation
+import com.mapbox.navigation.core.MapboxNavigationProvider
+import com.mapbox.navigation.instrumentation_tests.R
+import com.mapbox.navigation.instrumentation_tests.activity.EmptyTestActivity
+import com.mapbox.navigation.instrumentation_tests.utils.MapboxNavigationRule
+import com.mapbox.navigation.instrumentation_tests.utils.idling.RouteAlternativesIdlingResource
+import com.mapbox.navigation.instrumentation_tests.utils.location.MockLocationReplayerRule
+import com.mapbox.navigation.instrumentation_tests.utils.readRawFileText
+import com.mapbox.navigation.instrumentation_tests.utils.runOnMainSync
+import com.mapbox.navigation.testing.ui.BaseTest
+import com.mapbox.navigation.testing.ui.utils.getMapboxAccessTokenFromResources
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import java.util.concurrent.TimeUnit
+
+class RouteAlternativesTest : BaseTest<EmptyTestActivity>(EmptyTestActivity::class.java) {
+
+    @get:Rule
+    val mapboxNavigationRule = MapboxNavigationRule()
+
+    @get:Rule
+    val mockLocationReplayerRule = MockLocationReplayerRule(mockLocationUpdatesRule)
+
+    private lateinit var mapboxNavigation: MapboxNavigation
+
+    @Before
+    fun setup() {
+        val routeAlternativesOptions = RouteAlternativesOptions.Builder()
+            .intervalMillis(TimeUnit.SECONDS.toMillis(10))
```

Comment:
```suggestion
            .intervalMillis(TimeUnit.SECONDS.toMillis(5))
```

always less execution time and shouldn't change much in terms of results :upside_down_face:

Reply:
and maybe add an internal backdoor to set this if we increase the minimum interval value

Reply:
The current value is set to the minimum - also happy to reduce the minimum. Noting that, the larger values cause idle resource timeouts.

I don't want the test to use an internal back door, because customers are also requesting ways to run integration tests for their alternative route solutions.